src/applications/static-pages/analytics/addButtonLinkListeners.js

```diff
@@ -0,0 +1,14 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+export default function addHomepageBannerListeners() {
+  const buttonLinks = document.querySelectorAll('a.usa-button');
+
+  buttonLinks.forEach(buttonLink => {
+    buttonLink.addEventListener('click', event => {
+      recordEvent({
+        event: 'cta-default-button-click',
+        buttonText: event.target.text,
+      });
+    });
+  });
+}
```

Comment:
@kelsonic yeah good call, I did a 1-1 event binding (instead of listening at the container level) because these buttons are pretty infrequent, but if it becomes more of a thing then we should use the container approach